helm/kube-image-keeper/values.yaml

```diff
@@ -218,6 +218,13 @@ registry:
     schedule: "0 0 * * 0"
     # -- If true, delete untagged manifests. Default to false since there is a known bug in **docker distribution** garbage collect job.
     deleteUntagged: false
+    image:
+      # -- Cronjob image repository
+      repository: bitnami/kubectl
```

Comment:
I kept the existing value for compatibility, but changing the image repository to `docker.io/bitnami/kubectl` would fix #275 out of the box.